src/hotspot/share/gc/g1/g1EvacFailure.cpp

```diff
@@ -178,17 +189,23 @@ class RemoveSelfForwardPtrHRChunkClosure : public G1HeapRegionChunkClosure {
   }
 
 public:
-  RemoveSelfForwardPtrHRChunkClosure(size_t* marked_words_in_regions,
-                                     uint worker_id) :
+  RemoveSelfForwardPtrHRChunkClosure(uint worker_id) :
     _g1h(G1CollectedHeap::heap()),
-    _marked_words_in_regions(marked_words_in_regions),
-    _worker_id(worker_id) {
+    _worker_id(worker_id),
+    _region_idx(),
```

Comment:
region_idx should be initialized to `_g1h->max_regions()` here, at least I can't find an initialization.